lib/qless/server/views/layout.erb

```diff
@@ -67,7 +67,7 @@
      * in a vein similar to that of Ruby's flash */
     var flash = function(message, t, duration) {
       var noty_id = noty({
-        text   : '<strong>' + message + '</strong>',
+        text   : $('<strong/>').text(message),
```

Comment:
do we need the slash here ?

Reply:
It's probably not strictly necessary, but most examples include it.

Reply:
Seems to produce the same output with and without slash 🤷‍♂️